Makefile

```diff
@@ -1,7 +1,8 @@
 COMMIT?=$(shell git describe HEAD)$(shell git diff --quiet || echo '+dirty')
 
 # Use linker flags to provide commit info
-COMMON_LDFLAGS=-X=github.com/foundriesio/fioctl/subcommands/version.Commit=$(COMMIT)
+VERSION_LDFLAGS=-X=github.com/foundriesio/fioctl/subcommands/version.Commit=$(COMMIT)
+COMMON_LDFLAGS=-v -linkmode=external $(VERSION_LDFLAGS)
```

Comment:
Is this `-V` needed in the release version?

Reply:
Yes, I want to see what our CI prints.
There is just a handful lines of additional output which gives a lot of context in case of a problem.